core/templates/dev/head/pages/exploration_player/AudioBarDirective.js

```diff
@@ -28,19 +28,108 @@ oppia.directive('audioControls', [
       },
       templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
         '/pages/exploration_player/' +
-        'audio_controls_directive.html'),
+        'audio_bar_directive.html'),
       controller: [
-        '$scope', 'AudioTranslationManagerService', 'AudioPlayerService',
-        'LanguageUtilService', 'AssetsBackendApiService',
+        '$scope', '$interval', 'AudioTranslationManagerService',
+        'AudioPlayerService', 'LanguageUtilService', 'AssetsBackendApiService',
         'AutogeneratedAudioPlayerService', 'PlayerPositionService',
+        'WindowDimensionsService', 'EVENT_ACTIVE_CARD_CHANGED',
         function(
-            $scope, AudioTranslationManagerService, AudioPlayerService,
-            LanguageUtilService, AssetsBackendApiService,
-            AutogeneratedAudioPlayerService, PlayerPositionService) {
+            $scope, $interval, AudioTranslationManagerService,
+            AudioPlayerService, LanguageUtilService, AssetsBackendApiService,
+            AutogeneratedAudioPlayerService, PlayerPositionService,
+            WindowDimensionsService, EVENT_ACTIVE_CARD_CHANGED) {
           // This ID is passed in to AudioPlayerService as a means of
           // distinguishing which audio directive is currently playing audio.
           var directiveId = Math.random().toString(36).substr(2, 10);
 
+          $scope.isAudioBarExpanded = false;
```

Comment:
$scope.audioBarIsExpanded (to be clear that it's not a function)

Reply:
done